src/main/java/com/faforever/client/chat/AbstractChatTabController.java

```diff
@@ -556,7 +556,7 @@ private String renderHtml(ChatMessage chatMessage, URL themeFileUrl, @Nullable I
     String timeString = timeService.asShortTime(chatMessage.getTime());
     html = html.replace("{time}", timeString)
         .replace("{avatar}", StringUtils.defaultString(avatarUrl))
-        .replace("{username}", login)
+        .replace("{username}", login + ":")
```

Comment:
needs to be done why I18N Sevice class and the String added to message.properties file. You may say this does not need to be translated but there are languages that work right to left in that case the : would be on the other end